### DISCUSSION TITLE
Legality of MS-like themes

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2352] [Q&A]
I'm working on a Win2K-ish (and 95-ish colored) LXQt theme (see screenshot below). While other DEs have done this kind of thing before and seemingly not faced problems, for example the Redmond GTK2 engine, I wanted to get peoples' ideas about just how legal this is. It's aping Microsoft's trade dress after all.

I do think a Win9x/2K theme and palette would go a long way toward making LXQt more accessible to more users, which is also the idea behind the Adwaita theme set I'm working on at the same time. But it's not F/OSS. Would we be in legal trouble if we included a theme like this in the base set?

![LXQtRedmond2000](https://user-images.githubusercontent.com/67122280/209264020-2646739c-e6d5-41ac-93cc-79ee61d479ad.png)

**Comment by Butterfly** [comment]
I've wondered about that the last few years because somewhere I've seen at least one that outright calls itself something unmistakably Windows. :laughing: I tried a quick search. Started out looking for Windows themselves offering content because of their public "adoption" of GitHub on some level.

This popped up, instead:

[B00merang-Project / Windows-10](https://github.com/B00merang-Project/Windows-10)

They've addressed some extensive aspects of their own work, e.g. multiple areas where incompatibility painfully always pops up for developers.

For this Windows-10 one, it's About statement simply says, "Windows 10 Light theme for **_Linux_** (GTK)." In pondering whether to suggest it here, that account has been up a while, thus has not had a takedown request. Hopefully it could be implied they may have successfully addressed the obvious legal aspects of this topic. I thought maybe you could chat pro to pro if it seems interesting. :smile:

**Reply by edwardp3** [reply]
There are actually several such themes at https://store.kde.org/browse?cat=446&ord=latest.
